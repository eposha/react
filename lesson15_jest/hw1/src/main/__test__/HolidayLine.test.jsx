import React from "react";
import { shallow } from "enzyme";
import HolidayLine from "../HolidayLine";

describe("<HolidayLine/ >", () => {
  it("should display HolidayLine", () => {
    const wrappedComponent = shallow(<HolidayLine />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});
