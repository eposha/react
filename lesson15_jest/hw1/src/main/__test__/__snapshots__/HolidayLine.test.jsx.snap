// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockHours/ > should display BlockHours 1`] = `
Array [
  <div
    className="block-hour"
    key="0"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="1"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="2"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="3"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="4"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="5"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="6"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="7"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="8"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="9"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="10"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="11"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="12"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="13"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="14"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="15"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="16"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="17"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="18"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="19"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="20"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="21"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="22"
    onClick={[Function]}
  />,
  <div
    className="block-hour"
    key="23"
    onClick={[Function]}
  />,
]
`;

exports[`<HolidayLine/ > should display HolidayLine 1`] = `
<div
  className="holiday-wrapper"
>
  <div
    className="holiday"
  >
    <div
      className="holiday__day"
      key="0"
    />
    <div
      className="holiday__day"
      key="1"
    />
    <div
      className="holiday__day"
      key="2"
    />
    <div
      className="holiday__day"
      key="3"
    />
    <div
      className="holiday__day"
      key="4"
    />
    <div
      className="holiday__day"
      key="5"
    />
    <div
      className="holiday__day"
      key="6"
    />
  </div>
</div>
`;
